search/search_repository.go

```diff
@@ -400,7 +401,7 @@ var searchKeyMap = map[string]string{
 
 func (q Query) determineLiteralType(key string, val string) criteria.Expression {
 	switch key {
-	case workitem.SystemAssignees, workitem.SystemLabels:
+	case workitem.SystemAssignees, workitem.SystemLabels, workitem.SystemBoardcolumns, "Board":
```

Comment:
Why "Board"? Can't we create a `workitem.` constant?

Reply:
Fixed. The thing here is that is different. The "board" is not a direct workitem field. But I added it anyway.